supervisely/api/image_api.py

```diff
@@ -2736,6 +2736,84 @@ def add_tag_batch(
             if progress_cb is not None:
                 progress_cb(len(batch_ids))
 
+    def add_tags_batch(
+        self,
+        image_ids: List[int],
+        tag_ids: Union[int, List[int]],
+        values: Optional[Union[str, int, List[Union[str, int]]]] = None,
```

Comment:
values: Optional[Union[str, int, List[Optional[Union[str, int]]]]] = None
---------------------------------------^^^^^-----------------------------
For None tags?